public-api/src/test/java/org/pmiops/workbench/cdr/DataBrowserControllerTest.java

```diff
@@ -239,6 +284,61 @@ public void testGetDbDomains() throws Exception{
         );
     }
 
+    @Test
+    public void testConceptSearchWithEmptyQuery() throws Exception{
+        saveData();
+        ResponseEntity<ConceptListResponse> response = dataBrowserController.searchConcepts(new SearchConceptsRequest().query(""));
+        List<Concept> concepts = response.getBody().getItems().stream().map(TO_CLIENT_CONCEPT).collect(Collectors.toList());
+        assertThat(concepts)
+                .containsExactly(CONCEPT_1, CONCEPT_2, CONCEPT_3, CONCEPT_4, CONCEPT_5, CONCEPT_6)
+        ;
+    }
+
+
+    @Test
+    public void testConceptSearchEmptyCount() throws Exception{
+        // Exact concept search was not tested. Match in tests was replaced by like and like > 0.0 was not possible to test
```

Comment:
Comment this better like : 
" We can't test limiting to count > 0 with a concept name search because the match function does not work in hibernate. So we make several concepts with same concept code and one with count 0.   The limit > 0 works the same weather it is code or  name  match.  "